package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vant-react",
-  "version": "0.2.8",
+  "version": "0.2.9",
```

Comment:
this is a breaking change, should update the minor version instead of patch version

Reply:
0.3.0